docker/requirements.txt

```diff
@@ -0,0 +1,22 @@
+PyYAML
+bandit
+black
+cmakelint
+cpplint
+deprecated
+docformatter
+flawfinder
+isort
+lizard
+mypy
+pycodestyle
+pydocstyle
+pyflakes
+pylint
+pylint-django<2.0
+statick
+statick-md
+tabulate
+xmltodict
+yamllint
+yapsy
```

Comment:
It is less than ideal to have an exact copy of a file one directory up here.  But it's needed, at the moment, due to how Docker build contexts works.  I create an issue already to upgrade the use of the docker-build-and-push action to a new version that allows specifying multiple build contexts.   I tried to update it in this PR, but after spending 30 minutes fiddling with it, created the new issue to deal with it separately.

Reply:
I take this back.  It turns out that we want the requirements for the Docker image to be different than the one for statick proper, because we're adding the full suite of statick modules for use in the image.